docs/topics/basic-syntax.md

```diff
@@ -229,7 +272,7 @@ See [classes](classes.md) and [objects and instances](object-declarations.md).
 
```

Comment:
The line before that:
See [classes](classes.md) and [objects and instances](object-declarations.md). 
could benefit from the same treatment you have given the links in the previous section.

For example, we talk about primary constructors but not about secondary constructors - we could lead users to classes to read more about that.